src/admin/settings/settings.scss

```diff
@@ -0,0 +1,25 @@
+.form-group{
```

Comment:
@mkruszynski Either prefix this code so it just works in this settings app, or [add it to forms.scss](https://github.com/motech/motech-ui/blob/master/src/common/form/form.scss#L125) (where it might screw other things up, but we can catch those and shape up the codebase together)